app/controllers/builds_controller.rb

```diff
@@ -131,6 +131,10 @@ def modified_time
     end
   end
 
+  def resend_status
+    BuildStateUpdateJob.enqueue(params[:id])
```

Comment:
Does it need to respond with anything? 200 OK at least...

Reply:
heh good point